sycl/source/detail/queue_impl.hpp

```diff
@@ -403,7 +403,7 @@ class queue_impl {
   /// Stores a USM operation event that should be associated with the queue
   ///
   /// \param Event is the event to be stored
-  void addUSMEvent(const event &Event);
+  void addSharedEvent(const event &Event);
```

Comment:
Minor: could you update the comment above as well?

Reply:
The comment is updated, maybe in a later commit?  It reads 
```
/// addSharedEvent - queue_impl tracks events with weak pointers
/// but some events have no other owner. In this case,
/// addSharedEvent will have the queue track the events via a shared pointer.
```

Reply:
I see the comment you mentioned in queue_impl.cpp, but the one in queue_impl.hpp still mentions USM.

Reply:
d'oh!  Got it.